src/pages/careers/besquare/components/sections/_faq.tsx

```diff
@@ -10,7 +10,7 @@ import {
     TextContainer,
 } from '../../static/style/_faq'
 import { Accordion, AccordionItem } from 'components/elements'
-import { Show } from 'components/containers'
+import { Desktop } from 'components/containers/visibility'
```

Comment:
This can be imported directly from `components/containers`

Reply:
ok